sdk/identity/identity/test/manual-integration/AzureFunctions/INSTRUCTIONS.md

```diff
@@ -0,0 +1,86 @@
+# Testing Identity in Azure Functions
+
+## Configuration
+
+Before you begin, you must have the following:
+
+- An Azure account with an active subscription
+- The Azure Functions Core Tools version 3.x
+- Node.js, Active LTS version
+- One of the following tools for creating Azure resources:
+  - Azure CLI version 2.4 or later
+  - Azure PowerShell version 5.0 or later
+
+## Prerequisites
+
+Verify your prerequisites, which depend on whether you are using Azure CLI or Azure PowerShell for creating Azure resources
+
+For Azure CLI,
+
+1. In a terminal or command window, run func --version to check that the Azure Functions Core Tools are version 3.x.
```

Comment:
```suggestion
1. In a terminal or command window, run `func --version` to check that the Azure Functions Core Tools are version 3.x.
```

Is this the correct command? The commands like this one look better within these quotes.